test/javascripts/turbolinks_test.coffee

```diff
@@ -163,6 +174,12 @@ describe 'Turbolinks', ->
           assert(Turbolinks.fullPageNavigate.called, 'Should perform a full page refresh.')
           done()
 
+      it 'changed assets do not update the browser history stack', (done) ->
```

Comment:
My preference would be to move this out of the anonymous asset tests, and create a `describe` specifically for hash change events.  I'm okay with that happening outside of this PR.